src/algorithms/crypto/randomness.md

```diff
@@ -2,15 +2,28 @@
 title: "Randomness"
 ---
 
+TODO: clean up stale .id/.go files
+
 {{<label randomness>}}
 
 Randomness is used throughout the protocol in order to generate values and extend the blockchain.
-Random values are drawn from the {{<sref ticket_chain>}} and appropriately formatted for usage.
+Random values are drawn from a {{<sref drand>}} beacon and appropriately formatted for usage.
 We describe this formatting below.
 
-## Encoding On-chain data for randomness
+## Encoding Random Beacon randomness for on-chain use
+
+Entropy from the drand beacon can be harvested into a more general data structure: a `BeaconEntry`, defined as follows:
 
-Entropy from the ticket-chain can be combined with other values to generate necessary randomness that can be
+```go
+type BeaconEntry struct {
+    // Drand Round for the given randomness
+    Round       uint64
+    // Drand Signature for the given Randomness
+    Data   []byte
```

Comment:
This was a point from @whyrusleeping (as per lotus impl of the beacon), the idea being to adopt a more general name in case the beacon ever changes and no longer relies on a signature per se. 

I am very agnostic here. I think I tend to agree that the change would be large enough that leaving it as sig is simpler for now...